docs/developer/checkout.mdx

```diff
@@ -11,6 +11,11 @@ The below process describes the key milestones in the checkout process flow in S
 
 The code snippets included in this section may be run in [Playground](getting-started/graphql) or your preferred HTTP client.
 
+### Why there is no cart model?
+
+Unlike most of other ecommerce platforms, Saleor has no separate model for a cart. Instead, we decited to merge it with checkout. Main reason was providing additional address context, required to calculate taxes in different countries.
+Checkout provides interface for usual cart operations like managing adding products or promo codes. 
```

Comment:
```suggestion
Checkout provides the interface for usual cart operations like adding products or promo codes. It can also be completed in almost any order, for example, saving a shipping address before adding any items.
```

⚠️ Please confirm the above is true.

Reply:
That's an unusual requirement. What happens if I add a product, give my shipping address, and then remove that product? Shouldn't checkout only _require_ the shipping address if there are any shippable products and skip it otherwise, even if one was submitted?